libbpf-tools/Makefile

```diff
@@ -222,21 +223,21 @@ $(LIBBPF_OBJ): $(wildcard $(LIBBPF_SRC)/*.[ch]) | $(OUTPUT)/libbpf
 
 $(FSSLOWER_ALIASES): fsslower
 	$(call msg,SYMLINK,$@)
-	$(Q)ln -f -s $^ $@
+	$(Q)ln -f -s $(APP_PREFIX)$^ $@
```

Comment:
What is `APP_PREFIX` ?

Reply:
This is a variable that can be defined to prepend a prefix to the app binary names (e.g. `APP_PREFIX=bpf-`).

Reply:
I mean, should we add something like:
```makefile
APP_PREFIX ?= bpf-
```

Reply:
If you wanna make it the default then yes.

Reply:
I'd probably leave it as empty by default, for cases when people build locally from source